govuk_frontend_jinja/templates/components/details/macro.html

```diff
@@ -1,5 +1,5 @@
 {% macro govukDetails(params) %}
-  <details {%- if params.id %} id="{{params.id}}"{% endif %} class="govuk-details {%- if params.classes %} {{ params.classes }}{% endif %}" data-module="govuk-details" {%- for attribute, value in (params.attributes.items() if params.attributes else {}.items()) %} {{attribute}}="{{value}}"{% endfor %} {{- " open" if params.open }}>
+  <details {%- if params.id %} id="{{params.id}}"{% endif %} class="govuk-details {%- if params.classes %} {{ params.classes }}{% endif %}" data-module="govuk-details" {%- for attribute, value in (params.attributes.items() if params.attributes else {}.items()) %} {{attribute}}="{{ value | lower }}"{% endfor %} {{- " open" if params.open }}>
```

Comment:
As discussed on Slack, the `value | lower` there could potentially break things. Need instead to preprocess the params from govuk-uk before they reach the templates and lowercase the booleans at that point, not here.

Reply:
Just this and another bump to the latest bug fix version of `govuk-frontend`, then this should be good to go.

Reply:
Raised upstream PR to resolve at source https://github.com/alphagov/govuk-frontend/pull/2064